app/views/sephora_style_guide/home/_icons.slim

```diff
@@ -123,7 +123,55 @@ ruby:
     {
       "name": "PopStation SMS",
       "class": "pop-sms"
-    }
+    },
+    {
+      "name": "Awaiting Collection",
+      "class": "awaiting-collection"
+    },
+    {
+      "name": "Completed",
+      "class": "completed"
+    },
+    {
+      "name": "Customs On Hold",
+      "class": "customs-on-hold"
+    },
+    {
+      "name": "Delivery Attempted",
+      "class": "delivery-attempted"
+    },
+    {
+      "name": "Delivery Failed",
+      "class": "delivery-failed"
+    },
+    {
+      "name": "In Transit",
+      "class": "in-transit"
+    },
+    {
+      "name": "Invalidated",
+      "class": "invalidated"
+    },
+    {
+      "name": "Order Placed",
+      "class": "order-placed"
+    },
+    {
+      "name": "Packing Order",
+      "class": "packing-order"
+    },
+    {
+      "name": "Processing",
+      "class": "processing"
+    },
+    {
+      "name": "Return Requested",
+      "class": "return-requested"
+    },
+    {
+      "name": "With Driver",
+      "class": "with-driver"
+    },
```

Comment:
is there a reason why you chose a subset of the states?

Reply:
No reason. I chose the most general. Some states share the same icon. Not sure if it would be more preferable but alternatively i can
1. add all the states even though the have the same icon
2. rename the state to something more general and let the frontend handle the mapping of state to icon

Number 1 defeats the idea of extensibility since if we change the order state, we need to change the classes. Number 2 may give a bit of vagueness and add extra logic to the frontend.

Reply:
Sure we can have a name for each icon then map the states to the icon used.
```
{
van-pending: ['in_transit', 'with_driver'...]
credit-card-clock: ['payment_pending']
...
}
```